pkgs/applications/misc/gcfflasher/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, pkgs
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcfflasher";
+  version = "4.0.3-beta";
+
+  src = fetchFromGitHub {
+    owner = "dresden-elektronik";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-m+iDBfsHo+PLYd3K8JaKwhIXcnj+Q8w7gIgmHp+0plk=";
+  };
+
+  nativeBuildInputs = with pkgs; [ gcc pkg-config libgpiod ];
+
+  buildPhase = ''
+    ./build_posix.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv GCFFlasher $out/bin
+  '';
+
+  meta = with lib; {
+    description = "CFFlasher is the tool to program the firmware of dresden elektronik's Zigbee products";
+    license = licenses.bsd3;
+    maintainers = [ maintainers.fleaz ];
+  };
```

```suggestion
  meta = with lib; {
    description = "CFFlasher is the tool to program the firmware of dresden elektronik's Zigbee products";
    homepage = "...";
    license = licenses.bsd3;
    maintainers = with maintainers; [ fleaz ];
  };
```

- Adding the homepage makes it easier to navigate to the upstream project and browse changes.
- Using `with maintainers;` makes the maintainers list less noisy and easier to change.